src/buildHTML.js

```diff
@@ -1473,6 +1501,230 @@ groupTypes.accent = function(group, options) {
+                        body.depth + braceBody.height + 3 * ruleWidth);
+
+    let vlist = buildCommon.makeVList([
+        {type: "elem", elem: body, shift: 0},
+        {type: "elem", elem: braceBody, shift: braceShift},
+    ], "individualShift", null, options);
+
+    // In the vlist above, the brace takes its width from the subject matter.
+    // Don't put the optional note in that vlist. The note might set the width.
+    // Instead, put the note, if any, in a parent vlist.
+
+    if (haveNote) {
+        let noteShift;
+        if (group.value.isOver) {
+            noteShift = braceShift - braceBody.height - 4 * ruleWidth;
+        } else {
+            noteShift = braceShift + noteReset.height + 4 * ruleWidth;
+        }
+
+        const vSpan = makeSpan(["mord",
+            (group.value.isOver ? "mover" : "munder")],
+            [vlist], options);
+
+        vlist = buildCommon.makeVList([
+            {type: "elem", elem: vSpan, shift: 0},
+            {type: "elem", elem: noteReset, shift: noteShift},
+        ], "individualShift", null, options);
+    }
+
+    return makeSpan(["mord", (group.value.isOver ? "mover" : "munder")],
+        [vlist], options);
+};
+
+groupTypes.accentunder = function(group, options) {
+    // Treat under accents much like underlines.
+    const innerGroup = buildGroup(group.value.body, options);
+
+    const ruleWidth = fontMetrics.metrics.defaultRuleThickness /
+        options.style.sizeMultiplier;
+
+    const accentBody = stretchy.svgSpan(group, options);
+    const accentShift = innerGroup.depth + 2 * ruleWidth + accentBody.height;
+
+    // Generate the vlist, with the appropriate kerns
+    const vlist = buildCommon.makeVList([
+        {type: "elem", elem: innerGroup, shift: 0},
+        {type: "elem", elem: accentBody, shift: accentShift},
+    ], "individualShift", innerGroup.height, options);
+
+    return makeSpan(["mord", "accentunder"], [vlist], options);
+};
+
+groupTypes.strikeThru = function(group, options) {
+    const innerGroup = buildGroup(group.value.body, options);
+    const style = options.style;
+
+    if (group.value.label === "\\sout") {
+        const line = makeSpan(
+            [style.reset(), Style.TEXT.cls(), "stretchy", "sout"]);
+        line.height = fontMetrics.metrics.defaultRuleThickness /
+            style.sizeMultiplier;
+        line.maxFontSize = 1.0;
+
+        const lineShift = -0.5 * style.metrics.xHeight;
+        const vlist = buildCommon.makeVList([
+            {type: "elem", elem: innerGroup, shift: 0},
+            {type: "elem", elem: line, shift: lineShift},
+        ], "individualShift", null, options);
+        return makeSpan(["mord"], [vlist], options);
+
+    } else {
+        //  \cancel, \bcancel, \xcancel
+        let node = makeSpan([], [innerGroup], options);
+
+        // buildCommon.makeVList does not work well with a tall image span.
+        // So work out vertical placement here.
```

Comment:
When I pass a height much greater than 1.0 em to `buildCommon.makeVList`, it would respond to `maxFontSize` in a curious way. When given a low value of `maxFontSize`, `makeVList` would create a span that acted as if it had a large value of negative margin. If I passed a large value of `maxFontSize`, `makeVList` would create a span with too much height. 

I never found a way to pass a value that was just right. I'm not sure it is possible. In the end, I found it more reliable to write code that I could understand.

I haven't looked at how `llap` works. I'll do that. But keep in mind that `\cancel` has to make a *background* image appear in front of its subject matter. That complicates things.